src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -260,3 +261,70 @@ by setting the parameter `elasticsearch.index.mailbox.name` to the name of your
 re-creates index upon restart
 
 _Note_: keep in mind that reindexing can be a very long operation depending on the volume of mails you have stored.
+
+## Solving cassandra inconsistencies
+
+By choosing the cassandra, it means denormalization is the common modeling pattern to be using. 
```

Comment:
Cassandra backend uses data duplication to workaround Cassandra query limitations. However, Cassandra is not doing transaction when writing in several tables, this can lead to consistency issues for a given piece of data.